api/v1alpha1/stage_types.go

```diff
@@ -704,17 +704,23 @@ type AnalysisRunArgument struct {
 // process.
 type VerificationInfo struct {
 	// ID is the identifier of the Verification process.
-	ID string `json:"id,omitempty" protobuf:"bytes,4,opt,name=id"`
+	//
+	// +kubebuilder:validation:Required
+	ID string `json:"id" protobuf:"bytes,4,opt,name=id"`
```

Comment:
I'm sure you're two steps ahead of me, but what's the rationale for validation on fields that are only populated by the system?

Reply:
The system would have alerted me that it was missing, and this bug would have never existed if it were already in place.